.changeset/happy-camels-obey.md

```diff
@@ -0,0 +1,14 @@
+---
+"@frontity/hooks": minor
+"@frontity/components": patch
+"@frontity/core": patch
+"@frontity/invariant": patch
```

Comment:
This should be 3 separate changesets instead of one because each change should have its own changeset:

1. Deprecate useInView hook
2. Switch to `react-intersection-observer` in Image
3. Add `@frontity/invariant` package.

Maybe 1 and 2 could use the same changeset because they are related, but 3 has nothing to do.

Apart from that, `@frontity/invariant` should be a major (from 0.1.1 to 1.0.0).

Reply:
got it 🙁 .